src/app/shared/dialog/facet-entries/facet-entries.component.html

```diff
@@ -43,19 +43,20 @@
             </td>
             <td scope="row" [ngSwitch]="facet.type" *ngIf="!multiselect">
               <span *ngSwitchCase="'STRING'">
-                <a [routerLink]="routerLink" [queryParams]="getQueryParams(queryParams | async, facet, entry)" queryParamsHandling="merge">
+                [attr.aria-label]="'SHARED.DIALOG.FACET_ENTRIES.ACCESSIBILITY.FILTER_BY' | translate | translate"
```

Comment:
Please remove double translate.
```suggestion
                [attr.aria-label]="'SHARED.DIALOG.FACET_ENTRIES.ACCESSIBILITY.FILTER_BY' | translate"
```